### ISSUE TITLE
Tubed not working; with log-file issues; Raspi4

### ISSUE THREAD

**Issue by lizbethmutterhunt** [anxdpanic/plugin.video.tubed#187]
After days with working with API-keys now on the latest alpha 19 matrix I get: "youtube error: check log for information". In e log into /storage/.kodi/temp/kodi.log finds [this](https://pasteboard.co/JOBpIun.png) issues. I changed google account, still the same problem. BTW: the ones not from my account or beeing searched work (liked at the moment, a.s.o). Any help appreciated!

Lizbeth

**Comment by anxdpanic** [comment]
Installing https://github.com/anxdpanic/script.module.tubed.api/releases/download/1.0.4/script.module.tubed.api-1.0.4+matrix.1.zip from zip should resolve the issue. The fix is waiting for review as well, so should be in the official repository this week or so.

**Comment by lizbethmutterhunt** [comment]
Hi anxdpanic, 

super, trooper, it works! success to me, just implement it for others, the new plugin is damn good and in cooperation with google, so there are no issues because of the missing ads. 

thx a lot!

Lizbeth

**Comment by lizbethmutterhunt** [comment]
...but, when playing I get a lotta leaks and buffering issues; trying to reduce video-quality in settings to 360p but videos still play in HD. how to change? missing: "ask for quality" in the right-clicck submenu at a title. still in progress or my fault? maybe the CPU is too slow, I tried with Kali linux and doing an benchmark with ubuntu groovy mate now; let's see if it works without interrupts.

**Comment by lizbethmutterhunt** [comment]
just  to update my issue: I tried now diverse platforms in background like the new Q4OS and Kali linux, with Q4OS and the sid-repo on it I developed a rc2, but crashing at bootup. Doing Kali linux again as it has a rc3, nevermind the cpu-usage. DE: XFCE4 on Q4OS I had Plasma updated from KDE4 originally, but as told: crashing at bootup when updating the sid repo to a nightly git, just rc1. 

I keep you on the info. 

Lizbeth
